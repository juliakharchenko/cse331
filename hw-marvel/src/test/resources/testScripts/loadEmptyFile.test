# Test the parser on a graph that is empty

LoadGraph emptyGraph empty.csv
ListNodes emptyGraph