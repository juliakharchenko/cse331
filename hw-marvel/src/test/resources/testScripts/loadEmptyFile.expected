# Test the parser on a graph that is empty

loaded graph emptyGraph
emptyGraph contains: